R/rhtmlCombinedScatter.R

```diff
@@ -278,18 +288,44 @@ LabeledScatter <- function(
     xIsDateTime <- isDateTime(X[1])
     yIsDateTime <- isDateTime(Y[1])
 
+    color.levels <- NULL
+    colorIsDateTime <- FALSE
```

Comment:
Should this be `color.is.date.time`?

Reply:
Yep, fixed in [0fb9127](https://github.com/Displayr/rhtmlCombinedScatter/pull/20/commits/0fb9127fa8f5c00fda22ae104889ec3eda6b033f)